modules/oe_translation_poetry/tests/Functional/ContactFieldsWidgetTest.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+namespace Drupal\Tests\oe_translation_poetry\Functional;
+
+use Drupal\Core\Entity\Entity\EntityFormDisplay;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\field\Entity\FieldStorageConfig;
+use Drupal\node\NodeInterface;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests the contact field widgets.
+ */
+class ContactFieldsWidgetTest extends BrowserTestBase {
+
+  /**
+   * The Node storage.
+   *
+   * @var \Drupal\node\NodeStorageInterface
+   */
+  protected $nodeStorage;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'system',
+    'oe_translation_poetry',
+    'node',
+    'user',
+    'field',
+    'text',
+    'options',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->nodeStorage = \Drupal::entityTypeManager()->getStorage('node');
+
+    foreach ($this->getTestCases() as $case) {
+      FieldStorageConfig::create([
```

Comment:
I think its better if we are consistent and either use the static ::create methods or call \Drupal::entityTypeManager()->getStorage manually, but not both.

Reply:
Yes, apart from setting the node storage as class property, let's use static everywhere.